src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ContextQuery/SyntaxTokenExtensions.cs

```diff
@@ -223,7 +223,7 @@ public static bool IsBeginningOfGlobalStatementContext(this SyntaxToken token)
                     return true;
 
                 if (token.Parent is FileScopedNamespaceDeclarationSyntax namespaceDeclaration && namespaceDeclaration.SemicolonToken == token)
```

Comment:
@CyrusNajmabadi is this line necessary at all?

Reply:
i have no idea.  did anything break? :)